server/src/main/java/org/apache/druid/metadata/SqlSegmentsMetadataManager.java

```diff
@@ -1035,7 +1035,7 @@ public void poll()
   @GuardedBy("pollLock")
   private void doPoll()
   {
-    log.debug("Starting polling of segment table");
+    log.info("Starting polling of segment table");
```

Comment:
maybe also add/update logs in this class so that we know how long the poll method took and how long it took to build the timeline / compute overshadowed segments.

Reply:
@AmatyaAvadhanula , please check if any further logs are needed in this class to help debug future issues.